README.md

```diff
@@ -39,6 +39,7 @@ On each run, nearup self-updates to the latest version.
 > Ubuntu users can install it with the following command:
 >
 > ```
+> sudo apt update
```

Comment:
Shall we also add it in [the first call to `sudo apt install`](https://github.com/near/nearup/pull/62/files#diff-04c6e90faac2675aa89e2176d2eec7d8R21-R23). 

Github doesn't let me suggest change out of the current changes in the PR :(

Reply:
I will add it there too (it is better to overshoot). For some reason, people fail exactly here :thinking: